conductor_api/src/holo_signing_service.rs

```diff
@@ -0,0 +1,33 @@
+use boolinator::Boolinator;
+use holochain_core_types::{
+    agent::AgentId, cas::content::AddressableContent, error::HolochainError,
+};
+
+pub fn request_signing_service(
+    agent_id: &AgentId,
+    payload: &String,
```

Comment:
do we want this to be a string? I feel like it's more appropriate to sign a byte array - or rather, it *has* to be, so do we do a utf8 conversion on the other side?

Reply:
I was using Strings for the payload elsewhere before. And currently we only sign entry addresses which are represented as Strings in our code case as well. So I wouldn’t like to introduce a change just here...